tests/events/events.go

```diff
@@ -40,7 +38,7 @@ func ExpectEvent(object k8sObject, eventType, reason string) {
 func DeleteEvents(object k8sObject, eventType, reason string) {
 	By("Expecting events to be removed")
 	virtClient, err := kubecli.GetKubevirtClient()
-	util.PanicOnError(err)
+	Expect(err).ToNot(HaveOccurred())
```

Comment:
I think it better to use:
ExpectWithOffset(1, err).ToNot(HaveOccurred())
like in line 49

Reply:
Just out of curiosity, could you please elaborate why `ExpectWithOffset` would be better?
Thanks in advance!

Reply:
As documented in gomega:
" Unlike` `Expect` and `Ω`, `ExpectWithOffset` takes an additional integer argument that is used to modify the call-stack offset when computing line numbers. It is the same as `Expect(...).WithOffset`.

This is most useful in helper functions that make assertions. If you want Gomega's error message to refer to the calling line in the test (as opposed to the line in the helper function) set the first argument of `ExpectWithOffset` appropriately."